### PR TITLE
Added deserialize fix

### PR DESCRIPTION
**Pull request by arrangeesh**

### CODE REVIEW

File: kwikapi/setup.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
```
Comment (jaswanth098) [inline kwikapi/setup.py]: why is this file added here?

Comment (jaswanth098) [inline kwikapi/setup.py]: This is already there why are we adding new file inside kwikapi folder

**Comment by jaswanth098** [comment]
@arrangeesh Please use black module to format the changed file
```
pip3 install black
black file.py
```

**Review by jaswanth098 — APPROVED** [review]

**Comment by IdavalapatiRamanjaneyulu** [comment]
In older version of KwikAPI, we pass protocol name to _deserialize_response method and inside we do get the protocol object.

Later the code changed such that _deserialize_response will get protocol object directly and it uses that object.

nferx_cluster is using _deserialize_response of old KwikAPI version. But it is failing for new version of KwikAPI as it is sending the protocol name (backward compatibility).

Now the code modified such that _deserialize_response will check the type and work accordingly.

**Review by IdavalapatiRamanjaneyulu — APPROVED** [review]
